l10n_br_account_product/models/product.py

```diff
@@ -43,3 +62,8 @@ class ProductTemplate(models.Model):
 
     service_type_id = fields.Many2one(
         'l10n_br_account.service.type', u'Tipo de Serviço')
+
+    product_estimated_taxes_percent = fields.Float(
```

Comment:
up @mileo would be cool if we could merge that large PR before recreating the 9.0 and 10.0 branches... Waiting for your feedback on @renatonlima remarks.